pkg/cloudprovider/validationwrapper.go

```diff
@@ -63,8 +63,8 @@ func (w *cachingValidationWrapper) Create(m *v1alpha1.Machine, mcd *cloud.Machin
 }
 
 // Delete just calls the underlying cloudproviders Delete
-func (w *cachingValidationWrapper) Delete(m *v1alpha1.Machine, mcd *cloud.MachineCreateDeleteData) error {
-	return w.actualProvider.Delete(m, mcd)
+func (w *cachingValidationWrapper) Cleanup(m *v1alpha1.Machine, mcd *cloud.MachineCreateDeleteData) (bool, error) {
```

Comment:
Comment should be `Cleanup just calls...`

Reply:
Done with https://github.com/kubermatic/machine-controller/pull/385/commits/90f5bfd702b525c9ccbf0da106bc5ae30a5c8a66